plugin-template/testPlugin.py

```diff
@@ -1,5 +1,5 @@
 import crpropa
-import myPlugin
+import build.myPlugin as myPlugin
```

Comment:
Is this really the best way? 
I mean, it makes sense to have simply `import myPlugin`, right? That is probably achieved with a better folder structure.

Reply:
we fixed this behavior in PR #422. It is not needed anymore to import the module from the build folder. This was only to the missing `__init__` file. 
I would suggest to keep it as `import myPlugin`